src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -2570,7 +2570,7 @@ class C
                 expectedIndentation: 8);
         }
 
-        [WpfFact, Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        [WpfFact(Skip = "test is no longer correct due to language changes"), Trait(Traits.Feature, Traits.Features.SmartIndent)]
```

Comment:
If the test is skipped, a follow-up should follow. It can take many forms: remove, adjust, test new scenarios enabled by the change. Deciding what to do will be a part of resolving that issue, unless we can do that in context of this change.

---
In reply to: [91585635](https://github.com/dotnet/roslyn/pull/15548#discussion_r91585635) [](ancestors = 91585635,91584597)